internal/server/v1/info/golden/bulk_variable_group_info/economy.json

```diff
@@ -0,0 +1,27 @@
+{
+  "data": [
+    {
+      "node": "dc/g/Economy",
+      "info": {
+        "childStatVars": [
+          {
+            "id": "Count_Product_MobileCellularSubscription_AsFractionOf_Count_Person",
+            "searchName": "Count of Product With Product Type is Mobile Cellular Subscription (Per Capita)",
+            "searchNames": [
+              "Count of Product With Product Type is Mobile Cellular Subscription (Per Capita)"
+            ],
+            "displayName": "Mobile Subscriptions Per Capita",
+            "definition": "md=Count_Person,mp=count,pt=Product,productType=MobileCellularSubscription"
+          }
+        ],
+        "childStatVarGroups": [
+          {
+            "id": "dc/g/EconomicActivity",
+            "specializedEntity": "Economic Activity"
+          }
+        ]
+      }
+    },
+    {}
```

Comment:
For my own understanding, the expected behavior is just to return empty {} for the node "" instead of throwing an error. Is that right?

Reply:
I updated it to be a non-empty string now. So for input without information, it returns the node id only.